go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/go-openapi/strfmt v0.19.3
 	github.com/go-openapi/validate v0.19.5
 	github.com/go-ozzo/ozzo-validation v3.5.0+incompatible // indirect
+	github.com/godbus/dbus/v5 v5.0.3
```

Comment:
Shouldn't this come with a go.sum change?

Reply:
I think `github.com/godbus/dbus/v5` is already a transitive dependency used by runc which is vendored into k/k:

```
$ go mod why -m github.com/godbus/dbus/v5
# github.com/godbus/dbus/v5
k8s.io/kubernetes/pkg/kubelet/cm
github.com/opencontainers/runc/libcontainer/cgroups/systemd
github.com/godbus/dbus/v5
``` 

`github.com/godbus/dbus/v5` is already included in go.sum: `https://github.com/kubernetes/kubernetes/blob/master/go.sum#L215-L216`.

I think because it's already included, there's no need for additional `go.sum` change.